apps/condo/domains/user/schema/AuthenticateUserWithPhoneAndPasswordService.js

```diff
@@ -6,15 +7,35 @@ const { WRONG_PHONE_FORMAT } = require('@condo/domains/common/constants/errors')
 const { normalizePhone } = require('@condo/domains/common/utils/phone')
 const { STAFF } = require('@condo/domains/user/constants/common')
 const { WRONG_CREDENTIALS } = require('@condo/domains/user/constants/errors')
-const { AUTH_COUNTER_LIMIT_TYPE } = require('@condo/domains/user/constants/limits')
+const { LOGIN_BY_PHONE_AND_PASSWORD_LIMIT_TYPE } = require('@condo/domains/user/constants/limits')
 const { USER_FIELDS } = require('@condo/domains/user/gql')
 const { User } = require('@condo/domains/user/utils/serverSchema')
 const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
 
 const redisGuard = new RedisGuard()
 
-const GUARD_WINDOW_SIZE_SEC = 60 * 60 // seconds
-const GUARD_WINDOW_LIMIT = 10
+const GUARD_DEFAULT_WINDOW_SIZE_SEC = 60 * 60 // seconds
+const GUARD_DEFAULT_WINDOW_LIMIT = 10
+
+/**
+ * @type {Record<string, { windowSizeSec: number, windowLimit: number }>}
+ *
+ * Possible values:
+ * 1. Change all
+ * { "ip": { windowSizeSec: 3600, windowLimit: 60 } }
```

Comment:
Make example more clear. "ip" key makes it missleading. Consider using "<ip>" or "x.x.x.x" to specify, that keys must be specific ips, not just the "ip" word itself

Reply:
ok, fixed